wallabyjs/wallabyjs.d.ts

```diff
@@ -0,0 +1,167 @@
+// Type definitions for WallabyJS
+// Project: http://wallabyjs.com
+// Definitions by: Andrew Connell <http://www.andrewconnell.com>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'wallabyjs' {
+  namespace wallabyjs {
```

Comment:
This would look simpler when you remove the wrapping namespace (and the `export =`) and simply export each interface individually.